### PR TITLE
allow null values for input-var

### PR DESCRIPTION
**Pull request by JamesYL**

### CODE REVIEW

File: messaging/src/main/java/com/ultimatesoftware/workflow/messaging/correlation/CorrelationDataUtils.java [hunk 1]
```diff
@@ -84,7 +84,7 @@ private static String getOrderedJsonString(DocumentContext documentContext, Stri
             // Since we are using JacksonJsonNodeJsonProvider we need to convert
             // the result of the JsonPath into the value we need
             JsonNode node = documentContext.read(expression);
-            if (node == null || node.isNull()) {
+            if (node == null) {
```
Comment (ChrisFlanTheMan) [inline messaging/src/main/java/com/ultimatesoftware/workflow/messaging/correlation/CorrelationDataUtils.java:87]: can we add a test for the `node == null` and `node.isNull()` scenarios? just to make sure that it behaves as expected

**Review by ChrisFlanTheMan — APPROVED** [review]
Thanks for adding such thorough tests!
